rhoas/provider.go

```diff
@@ -75,6 +78,13 @@ func Provider() *schema.Provider {
 func providerConfigure(ctx context.Context, d *schema.ResourceData) (interface{}, diag.Diagnostics) {
 	// Warning or errors can be collected in a slice type
 	var diags diag.Diagnostics
+
+	localizer, err := goi18n.New(nil)
+	if err != nil {
+		tflog.Error(ctx, err.Error())
+		os.Exit(1)
```

Comment:
Probably copy paste. That should never happen in terraform (it can happen in the CLI)

Reply:
Well, it's possible that the directory is moved right which I know won't happen anyway.

Things like this is why I would like asserts in go outside of tests, if we fail to create a localizer there is no need for nice go error handling if something has blown up.